addons/web/static/src/core/dialog/dialog.js

```diff
@@ -98,7 +98,7 @@ export class Dialog extends Component {
             ref: this.modalRef,
             elements: ".modal-content",
             handle: ".modal-header",
-            ignore: "button",
+            ignore: "button, input",
```

Comment:
Hello, @odoo/rd-framework-js , do you agree with this change?

Reply:
Looks reasonable (not sure about having inputs in the header though, but if that's what we want...)